auth/initialConf.py

```diff
@@ -113,16 +113,14 @@ def create_permissions():
         permission_dict_helper('ro_user', "/auth/user/(.*)", "GET"),
         permission_dict_helper('all_pap', "/auth/pap/(.*)", "(.*)"),
         permission_dict_helper('ro_pap', "/auth/pap/(.*)", "GET"),
-        permission_dict_helper('all_x509_ca', "/x509/v1/ca/(.*)", "(.*)"),
-        permission_dict_helper('ro_x509_ca', "/x509/v1/ca/(.*)", "GET"),
-        permission_dict_helper('all_x509_certificates', "/x509/v1/certificates/(.*)", "(.*)"),
-        permission_dict_helper('ro_x509_certificates', "/x509/v1/certificates/(.*)", "GET"),
```

Comment:
Can these routes really be removed? Won't it impact access to the x509 certificate component?

Reply:
No! =O